evalml/pipelines/components/transformers/feature_selection/feature_selector.py

```diff
@@ -70,3 +70,29 @@ def fit_transform(self, X, y=None):
             pd.DataFrame: Transformed data.
         """
         return self.fit(X, y).transform(X, y)
+
+    def _handle_partial_dependence_fast_mode(self, X, pipeline_parameters):
+        """Updates pipeline parameters to not drop any features based off of feature importance.
+
+        Note:
```

Comment:
A style mega nit is that we don't need this in its own section. We can mirror other parts of the docstring here by just keeping it as its own unmarked separate paragraph.

Reply:
@eccabay Do you mean like this?
![image](https://user-images.githubusercontent.com/64278226/200063668-9711c4dc-e869-40ac-800b-c7a4bc06b326.png)